platform-ajax/src/test/java/com/softicar/platform/ajax/event/AjaxDomEventWithExceptionTest.java

```diff
@@ -44,4 +46,18 @@ public void test() {
 		// assert text that was appended before the Exception occurred is displayed
 		assertEquals("Hello", getText(output));
 	}
+
+	@Test
+	public void testWithNonSofticarUserException() {
+
```

Comment:
should we also test that the document was properly created?
```suggestion
// check that the document was created properly
assertEquals("", getText(output));
```